Anatolie/src/main/java/AreaOfCircle.java

```diff
@@ -0,0 +1,10 @@
+public class AreaOfCircle {
+    public static void main(String[] args) {
+
+        double radius;
+        radius = 3;
+        double Area;
```

Comment:
Please follow the naming convention (local variable name - first letter lowercase)

Reply:
Done